Add a news article for the SR20201 fourth league session (#268)
srobo/website: _posts/2021-04-24-sr2021-final-league-session.md

---
layout: news
title: The SR2021 Final League Session comes to a close
---

With the final 36 matches of the SR2021 League complete, team MAI from Gymnasium
Markt Indersdorf head up the leaderboard. They have managed to knock the teams from
Hills Road Sixth Form College out of the top spot. We saw great perfomances from all
teams as well as some innovative new strategies, the finals are shaping up to be a
close call!

Module Ⅳ totally redisigned the arena, while keeping the same game mechanics, meaning teams
have had to make some large changes to their strategy to ensure continued success. Our new
larger arena includes more territories, moveable obstacles, and new kinds of territories
worth varying points.

> Now the league stage of the competition is over the Knockouts await.

Reviewer: ```suggestion
Now the league stage of the competition is over, the Knockouts await.
```

Reviewer 2: Don't think this is needed?

Author: Yeah not sure this is needed

There will be one play-in match between the bottom two teams in order to reduce
them to one team followed by an ordinary single-elimination knockout of the 16

> remaining teams. Pairings are arranged such that the higher the position in the league

Reviewer: ```suggestion
remaining teams. Pairings are arranged such that the higher your position in the league
```

> remaining teams. Pairings are arranged such that the higher the position in the league
> the easier your path to the final.

Reviewer 2: or maybe? Think it's fine as-is though.
```suggestion
remaining teams. Pairings are arranged such that the higher a team's position in the league
the easier their path to the final.
```

Author: 4d48374
